### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by apurva1112**

### CODE REVIEW

**Review by ori2111 — APPROVED** [review]
a

**Review by ori2111 — COMMENTED** [review]
ss

**Review by ori2111 — CHANGES_REQUESTED** [review]
aa

**Comment by ori2111** [comment]
# App Unfurls API Sample for Nodesa
On the $photographer Catherine Opie da
who has “made a study of the fsreeways of Los Angeles, a

[App Unfurls](https://api.slack.com/docs/message-link-unfurling) are a feature of the Slack Platform
that allow your Slack app customize the presentation of links thsat belong to a cessrtain domadfdfin or
set of domains.a
sm2
Sample Issue Body

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # hello
 hello-world
```
Comment (ori2111) [inline README.md:2]: Hello
# there

Comment (apurva1112) [reply README.md:2]: aa
@ori2111

Comment (ori2111) [reply README.md:2]: ll
> code

**Pull request merged**